fixtures/parallelizer/remote.py

```diff
@@ -13,12 +14,12 @@
 
 class SlaveManager(object):
     """SlaveManager which coordinates with the master process for parallel testing"""
-    def __init__(self, config, slaveid, base_url, zmq_endpoint):
+    def __init__(self, config, slaveid, appliance_config, zmq_endpoint):
         self.config = config
         self.session = None
         self.collection = None
         self.slaveid = conf.runtime['env']['slaveid'] = slaveid
-        self.base_url = conf.runtime['env']['base_url'] = base_url
+        self.appliance_config = conf.runtime['env']['appliances'][0] = appliance_config
```

Comment:
**optional** off hand i fear this might create a wrong structure in yaycl/lya - please verify the type of `conf.runtime['env']['appliances']`

Reply:
I tested this locally w/ pytest parallel runs -- e.g. `py.test --appliance 1.2.3.4 --appliance 3.4.5.6` and all was well. In fact I was hitting a TypeError here before adding the [0], 'appliances' is expected to be a list of dicts

Reply:
fabulous, thanks 👍